module-system/module-implementations/sov-attester-incentives/src/lib.rs

```diff
@@ -0,0 +1,206 @@
+#![deny(missing_docs)]
+#![doc = include_str!("../README.md")]
+
+/// Call methods for the module
+pub mod call;
+
+/// Methods used to instantiate the module
+pub mod genesis;
+
+#[cfg(test)]
+mod tests;
+
+#[cfg(feature = "native")]
+#[allow(missing_docs)]
+pub mod query;
+
+use std::marker::PhantomData;
+
+use borsh::{BorshDeserialize, BorshSerialize};
+use call::Role;
+use sov_modules_api::{Context, Error};
+use sov_modules_macros::ModuleInfo;
+use sov_rollup_interface::zk::{
+    StoredCodeCommitment, ValidityCondition, ValidityConditionChecker, Zkvm,
+};
+use sov_state::{Storage, WorkingSet};
+
+/// Configuration of the attester incentives module
+pub struct AttesterIncentivesConfig<
+    C: Context,
+    Vm: Zkvm,
+    Cond: ValidityCondition,
+    Checker: ValidityConditionChecker<Cond>,
+> {
+    /// The address of the token to be used for bonding.
+    pub bonding_token_address: C::Address,
+    /// The address of the account holding the reward token supply
+    pub reward_token_supply_address: C::Address,
+    /// The minimum bond for an attester.
+    pub minimum_attester_bond: u64,
+    /// The minimum bond for a challenger.
+    pub minimum_challenger_bond: u64,
+    /// A code commitment to be used for verifying proofs
+    pub commitment_to_allowed_challenge_method: Vm::CodeCommitment,
+    /// A list of initial provers and their bonded amount.
+    pub initial_attesters: Vec<(C::Address, u64)>,
+    /// The finality period of the rollup (constant)
+    pub rollup_finality_period: u64,
+    /// The current maximum attested height
+    pub maximum_attested_height: u64,
+    /// The light client finalized height
+    pub light_client_finalized_height: u64,
+    /// The validity condition checker used to check validity conditions
+    pub validity_condition_checker: Checker,
+    /// Phantom data that contains the validity condition
+    phantom_data: PhantomData<Cond>,
+}
+
+/// The information about an attester's unbonding
+#[derive(BorshDeserialize, BorshSerialize, Clone, Debug)]
+pub struct UnbondingInfo {
+    /// The height at which an attester is allowed to withdraw their tokens
+    pub unbonding_initiated_height: u64,
```

Comment:
The comment and the variable name don't match here. Is this the height they started unbonding or the height they finish unbonding?